addons/point_of_sale/migrations/13.0.1.0.1/pre-migration.py

```diff
@@ -21,6 +21,22 @@
 ]
 
 
+def create_pos_picking_types(env):
+    """Create picking types according to former pos.config locations
+    """
+    configs = env["pos.config"].with_context(
+        active_test=False
+    ).search([]).filtered(
+        lambda x: x.stock_location_id
+        != x.picking_type_id.default_location_src_id
+    )
+    for config in configs:
+        config.picking_type_id = config.picking_type_id.copy({
+            "name": "{} - {}".format(config.picking_type_id.name, config.name),
```

Comment:
Maybe add "openupgrade" in the name to make explicit the création by openupgrade ?

Reply:
Yes, it's a good idea to be able to tell which ones were created by OU quickly.